### PR TITLE
Fsleyes

### PR DESCRIPTION
**Pull request by AbstractMethod**
pull-request for initial version of FSLeyes and supporting packages.

### CODE REVIEW

File: sci-biology/fsleyes-props/fsleyes-props-1.6.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+"
+S="${WORKDIR}/${MY_P}-93b5cadc15da10a2e6ffacf1304edbb1c9a8355a"
+
+src_compile() {
+	python setup.py build
+}
+
+src_install() {
+	python setup.py install --root "${D}"
+}
```
Comment (kiwifb) [inline sci-biology/fsleyes-props/fsleyes-props-1.6.4.ebuild]: If that's all you do in src_compile and src_install, remove them. distutils-r1 does a better job to do the right thing than what you put there. In fact I am pretty sure it is broken if you have multiple installation of python on your system.

File: sci-biology/fsleyes-widgets/fsleyes-widgets-0.6.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+"
+S="${WORKDIR}/${MY_P}-a5ca27c2aba66606040896026626c3afea8ae87e"
+
+src_compile() {
+	python setup.py build
+}
+
+src_install() {
+	python setup.py install --root "${D}"
+}
```
Comment (kiwifb) [inline sci-biology/fsleyes-widgets/fsleyes-widgets-0.6.6.ebuild]: As above.

File: sci-biology/fsleyes/fsleyes-0.26.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+S="${WORKDIR}/${P}-af42b7efba43f60b9cc6d63009d9819dda8979d8"
+
+src_compile() {
+	python setup.py build
+}
+
+src_install() {
+	python setup.py install --root "${D}"
+}
```
Comment (kiwifb) [inline sci-biology/fsleyes/fsleyes-0.26.2.ebuild]: As in the first comment again.

File: sci-biology/fslpy/fslpy-1.11.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+"
+S="${WORKDIR}/${P}-2575a60d0ba08c5cddc0eff83cbe6040b3efa994"
+
+src_compile() {
+	python setup.py build
+}
+
+src_install() {
+	python setup.py install --root "${D}"
+}
```
Comment (kiwifb) [inline sci-biology/fslpy/fslpy-1.11.1.ebuild]: Likewise.

**Comment by kiwifb** [comment]
Much better thank you.

**Comment by TheChymera** [comment]
@KoalaPulp Welcome to the team! I'm happy to see there's going to be more of us neuro* people contributing to Gentoo :)

I have also been working on FSLeyes for a while now, and this is unlikely to be easily addressed in one PR. There's a numer of packages we are working on, and some of them are more properly added in Gentoo main. I would suggest we close this omnibus PR, and pull each package separately while tracking the meta-discussion in [an issue](https://github.com/gentoo/sci/issues/894).

What I would generally comment on your PR is:

* You should test it (e.g. with `repoman full`). You should have noticed that your ebuilds fail to resolve dependencies, because they are marked Python 3* compatible yet depend on wxpython which is only Python 2* in main+science
* The upper bound dependency restrictions are because upstream is trying to be very stable. In truth they end up [introducing non-existing dependency stack conflicte](https://github.com/pauldmccarthy/fsleyes/issues/7) and will make bumping more difficult. You can safely ingore them.
* You should use indenting in `metadata.xml` analogous to what you see in existing ebuilds
* fslpy [conflicts with the current FSL](https://github.com/pauldmccarthy/fslpy/issues/1) so in addition to many other things this is blocked by [our current FSL version bump](https://github.com/gentoo/sci/pull/887).

**Comment by TheChymera** [comment]
As per https://github.com/gentoo/sci/issues/894 this can be closed by somebody who has the access required to close it.

**Pull request closed without merging**